.github/workflows/issue-lifecycle.yml

```diff
@@ -8,19 +8,18 @@ on:
 jobs:
   # If the created issue does not have a milestone attached to it, assign the issue to the backlog
   assign-milestone:
-    name: Assign Milestone to Issue
     runs-on: ubuntu-latest
     steps:
       - name: Assign the created issue to the backlog milestone if no milestone is defined.
         if: github.event.issue.milestone == null
         # Set environment
         env:
-          MILESTONE_ID: 5
+          MILESTONE_ID: 1
```

Comment:
Why change to `MILESTONE_ID: 1`? Is there a configuration option in the repo for GitHub actions to specify this should be 5: https://github.com/OSC/ondemand/milestone/5

Reply:
Oops this was my bad, `MILESTONE_ID: 5` is the correct value for the OSC/ondemand repo.